libs/api/src/main/java/de/cyface/api/model/Json.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2021 Cyface GmbH
+ *
+ * This file is part of the Cyface Data Collector.
+ *
+ * The Cyface Data Collector is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Cyface Data Collector is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the Cyface Data Collector. If not, see <http://www.gnu.org/licenses/>.
+ */
+package de.cyface.api.model;
+
+import java.util.Arrays;
+
+/**
+ * This class allows structuring data in the Json format without Json dependencies.
+ *
+ * @author Armin Schnabel
+ * @since 1.1.0
+ */
+public class Json {
```

Comment:
This allows us to generate Json or GeoJson without requiring a Json dependency in this library